README.md

```diff
@@ -11,39 +11,56 @@ Authors:
 - Florian Jäger
 - Frederik Stalschus
 
-## Project setup
+## Project Deployment
 
 1. Install [Docker](https://docs.docker.com/engine/install/) and run the docker daemon.
```

Comment:
Should we also require that docker compose is installed?

Reply:
Done.